lib/log_helper.py

```diff
@@ -34,15 +34,14 @@ def __init__(self, log_file_path=None, verbose=False, rotate_size=0):
         sh.setFormatter(logging.Formatter('%(message)s'))
         logger.addHandler(sh)
 
-        if log_file_path:
```

Comment:
If you're removing this `if`, then make the `log_file_path` required. Having a default value that breaks the function makes no sense.